configstore/backends/docker_secret.py

```diff
@@ -9,6 +9,10 @@ def __init__(self, secrets_path=SECRETS_PATH):
         self.secrets_path = secrets_path
 
     def get_config(self, key):
+        path = os.path.join(self.secrets_path, key)
 
-        with open(os.path.join(self.secrets_path, key)) as fd:
-            return fd.readline().strip()
+        try:
+            with open(path) as fd:
+                return fd.readline().strip()
+        except FileNotFoundError:
```

Comment:
We have projects on Python 2.7

![sorry](https://media.giphy.com/media/3ohA2ZD9EkeK2AyfdK/giphy.gif)